system/kube-system/charts/ingress/templates/ingress-controller-deployment.yaml

```diff
@@ -1,35 +1,47 @@
 apiVersion: extensions/v1beta1
-kind: DaemonSet 
-
+kind: Deployment
 metadata:
   name: ingress-controller
-
+  namespace: kube-system
+  labels:
+    app: ingress-controller
 spec:
```

Comment:
This is missing `replicas: 3` or something.
Also we need to ensure that only one pod is scheduled per master.
Specifying a `hostPort` should achieve this. E.g.: https://github.com/sapcc/helm-charts/blob/47b38505d2974cecb6d861286af3d56e201000d6/openstack/elk/templates/fluent-daemonset.yaml#L61-L62